Makefile

```diff
@@ -48,6 +48,7 @@ POSTCSS=npx postcss
 SASSFLAGS=-I node_modules/ --color --quiet-deps
 UGLIFYFLAGS=--compress --mangle "reserved=GrocyClass" --source-map "content=public/dist/grocy.js.map"
 RUNFLAGS=-S localhost:8000 -t public/ 
+YARNFLAGS=
```

Comment:
This always overrides the variable, even if I have set it in my environment. You should be able to leave this line out completely.

See also: https://www.gnu.org/software/make/manual/html_node/Environment.html#Environment

Case in point: `YARNFLAGS=--foobar make yarn.lock` runs without complaining; directly executing `yarn install --foobar fails`.

Reply:
I just realized the release build doesn't rebuild with the other flag probably because the yarn.lock file didn't change. Should we do something about that?

Reply:
Hmmm, it should validate if the timestamp on either the node_modules folder or package.json is newer than yarn.lock.

The last case doesn't happen unless you edit package.json manually; the first happens whenever some file is edited within node_modules (or at least should happen).

So basically, the logic is "if something changed between last install and this build step, validate".

But this doesn't catch a "edit -> install unrelated package -> release" scenario. Maybe it's better to just force the clean target for releases, but leave `${YARNFLAGS}` where it is, so I can get my changes get undone whenever I mess with node_modules.

Reply:
You wanna do the force clean target before release? Because I don't know how this would work with parallel builds. Feel free to push here if you want.